packages/shell-api/src/database.ts

```diff
@@ -159,6 +159,31 @@ export default class Database extends ShellApiWithMongoClass {
     ) || [];
   }
 
+  _isFLE2Collection(collections: Document[], index: number): boolean {
+    return (
+      !collections[index].name.startsWith('enxcol_.') &&
+      !!collections.find(coll => coll.name === `enxcol_.${collections[index].name}.esc`) &&
+      !!collections.find(coll => coll.name === `enxcol_.${collections[index].name}.ecc`) &&
+      !!collections.find(coll => coll.name === `enxcol_.${collections[index].name}.ecoc`)
```

Comment:
I think you can use `collections.some()` as a shorthand for `!!collections.find()`:

```suggestion
      collections.some(coll => coll.name === `enxcol_.${collections[index].name}.esc`) &&
      collections.some(coll => coll.name === `enxcol_.${collections[index].name}.ecc`) &&
      collections.some(coll => coll.name === `enxcol_.${collections[index].name}.ecoc`)
```